autoPyTorch/pipeline/components/setup/network_embedding/LearnedEntityEmbedding.py

```diff
@@ -19,69 +19,59 @@
 class _LearnedEntityEmbedding(nn.Module):
     """ Learned entity embedding module for categorical features"""
 
-    def __init__(self, config: Dict[str, Any], num_input_features: np.ndarray, num_numerical_features: int):
+    def __init__(self, config: Dict[str, Any], num_categories_per_col: np.ndarray, num_features_excl_embed: int):
         """
         Args:
             config (Dict[str, Any]): The configuration sampled by the hyperparameter optimizer
             num_input_features (np.ndarray): column wise information of number of output columns after transformation
```

Comment:
I think `num_input_features` should be replaced with `num_categories_per_col (np.ndarray): number of categories for categorical columns that will be embedded`